packages/components/ContextualMenu/src/ContextualMenuItem.types.ts

```diff
@@ -53,7 +54,7 @@ export interface ContextualMenuItemProps extends Omit<IPressableProps, 'onPress'
 }
 
 export interface ContextualMenuItemSlotProps {
-  root: React.PropsWithRef<ViewProps>;
+  root: React.PropsWithRef<ViewProps> & { acceptsKeyboardFocus?: boolean };
```

Comment:
is it legit to have a context menu that doesn't accept keyboard focus?  What's the scenario?

Reply:
No we want keyboard focus! This was to test focus behaviors for my next work item. I"ll remove it